pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -61,6 +61,23 @@ func (r *ReconcileStorageCluster) ensureStorageClasses(instance *ocsv1.StorageCl
 	return nil
 }
 
+func (r *ReconcileStorageCluster) newOBCStorageClass(initData *ocsv1.StorageCluster) *storagev1.StorageClass {
+	reclaimPolicy := corev1.PersistentVolumeReclaimRetain
+	retSC := &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("%s-ceph-retain-bucket", initData.Name),
+		},
+		Provisioner:   fmt.Sprintf("%s.%s/bucket", initData.Namespace, initData.ClusterName),
+		ReclaimPolicy: &reclaimPolicy,
+		Parameters: map[string]string{
+			"objectStoreName":      fmt.Sprintf("%s-object-store", initData.Namespace),
+			"objectStoreNamespace": initData.Namespace,
+			"region":               "us-east-1", // TODO: how to get the current region
```

Comment:
Can someone help me out here to get the current `region`, rather than a hardcoded region?
@leseb , in your sample OBC SC, you have provided `region` as `us-east-1`, but I guess we are trying to add the current region.

Reply:
Let's keep it hardcoded for now, we don't need to really expose this.

Reply:
It's more an S3 artifact to used to create the bucket and is present for historical reason AFAIR.

Reply:
Does it need to be an existing region then? Or can we just make up a name, so that it is not as confusing? :thinking: